cmd/apply.go

```diff
@@ -171,6 +171,10 @@ func RunApply(options *ApplyOptions) error {
 	}
 
 	logger.Always("The [%s] cluster has applied successfully!", newCluster.Name)
+	if path, ok := newCluster.Annotations[kubeconfig.ClusterAnnotationKubeconfigLocalPah]; ok {
+		logger.Always("To start using your cluster, you need to run")
+		logger.Always("  export KUBECONFIG='%s'", path)
```

Comment:
Perhaps the file could be appended to `KUBECONFIG`?

Reply:
@nrb the IMO value in the annotation should contain the file name
Should I rename from ClusterAnnotationKubeconfigLocalPath ClusterAnnotationKubeconfigLocalFile? 😉

Reply:
I think what @nrb is asking for is to suggest the user do:
```bash
export KUBECONFIG="${KUBECONFIG}:%s"
```
`KUBECONFIG` is a list of hierarchical paths:
See: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters#append-homekubeconfig-to-your-kubeconfig-environment-variable

Reply:
Yeah, what @stealthybox said. I might even go further and use `os.Setenv('KUBECONFIG', "${KUBECONFIG}:%s" and inform the user that the file was added.

Reply:
Ok! I will update the PR asap

Thanks @nrb for the clarification
@nrb os.SetEnv _only_ can change the environment definition inside your process and in child process (not in the parent process)

Reply:
Ahh, good catch. Yeah, in that case I'd support outputting the export command